### PR TITLE
A few minor documentation fixes

### PR DESCRIPTION
**Pull request by ZedThree**
- Get the version dynamically
- Fix some links
- Fix some other minor warnings when building the docs
- Add API autodoc files

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -62,25 +64,25 @@ def __getattr__(cls, name):
 master_doc = "index"
 
 # General information about the project.
-project = u"FreeGS"
-copyright = u"2017-2019, Ben Dudson"
-author = u"Ben Dudson"
+project = "FreeGS"
+copyright = "2017-2023, Ben Dudson"
```
Comment (bendudson) [inline docs/conf.py]:
```suggestion
copyright = "2017-2023, FreeGS authors"
```

File: docs/conf.py [hunk 2]
```diff
@@ -62,25 +64,25 @@ def __getattr__(cls, name):
...
+author = "Ben Dudson"
```
Comment (bendudson) [inline docs/conf.py:69]: Should we list everyone, or refer to the repository to see the authors?

**Review by bendudson — DISMISSED** [review]

File: docs/conf.py (hunk 2 above)
Comment (ZedThree) [reply docs/conf.py:69]: I count 14 contributors, probably a bit too many for here?

Comment (bendudson) [reply docs/conf.py:69]: Just "FreeGS authors"?

Comment (ZedThree) [reply docs/conf.py:69]: Done!

**Review by bendudson — APPROVED** [review]

**Pull request merged**